Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -1065,17 +1173,17 @@ void MatchTOF::doMatchingForTPC(int sec)
   return;
 }
 //______________________________________________
-int MatchTOF::findFITIndex(int bc)
+int MatchTOF::findFITIndex(int bc, const gsl::span<const o2::ft0::RecPoints>& FITRecPoints)
 {
-  if (mFITRecPoints.size() == 0) {
+  if (FITRecPoints.size() == 0) {
     return -1;
   }
 
   int index = -1;
   int distMax = 5; // require bc distance below 5
 
-  for (int i = 0; i < mFITRecPoints.size(); i++) {
-    const o2::InteractionRecord ir = mFITRecPoints[i].getInteractionRecord();
+  for (int i = 0; i < FITRecPoints.size(); i++) {
```

Comment:
this will lead to a warning about comparing signed and unsigned.

Reply:
move to "unsigned int i"